.circleci/config.yml

```diff
@@ -1,16 +1,46 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.8.5
+  architect: giantswarm/architect@0.8.9
 
 workflows:
   package-and-push-chart-on-tag:
     jobs:
       - architect/push-to-app-catalog:
-          name: "package and push kubernetes-prometheus-app chart"
+          name: push-to-default-app-catalog
+          app_catalog: "default-catalog"
+          app_catalog_test: "default-test-catalog"
+          chart: "prometheus-operator-app-chart"
+          filters:
+            # Trigger the job also on git tag.
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-app-catalog:
+          name: push-to-giantswarm-app-catalog
           app_catalog: "giantswarm-catalog"
           app_catalog_test: "giantswarm-test-catalog"
           chart: "prometheus-operator-app-chart"
-          # Needed to trigger job also on git tag.
           filters:
+            # Trigger the job also on git tag.
             tags:
               only: /^v.*/
+      
+      - architect/push-to-app-catalog:
+          name: push-to-app-catalog-master
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "prometheus-operator-app-chart"
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-app-catalog:
+          name: push-to-app-catalog-pr
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "prometheus-operator-app-chart"
+          filters:
+            branches:
+              ignore: master
```

Comment:
Having a single job to push to `control-plane-catalog` is enough.

By default circleci builds for every branches but not tag, so let's keep one job which is triggered for every branches and every tag.

i.e. 
```
          filters:
            tags:
              only: /^v.*/
```